### PR TITLE
Assignments improvements

### PR DESCRIPTION
**Pull request by santiagobasulto**
Student:

seleccion_122

Teacher:

seleccion_123

### CODE REVIEW

File: rmotr_sis/templates/courses/lecture_detail.html [hunk 1]
```diff
@@ -93,7 +102,15 @@
...
-                    <td>{{ status|capfirst }}</td>
+                    <td>
+                        {% if status == 'resolved' %}
+                          <i class="fa fa-check" style="color: #5cb85c"></i>
+                        {% elif status == 'failed' %}
+                          <i class="fa fa-times" style="color: red"></i>
+                        {% elif status == 'pending' or status == 'unsubmitted'%}
+                          <i class="fa fa-clock-o"></i>
+                        {% endif %}
+                    </td>
```
Comment (santiagobasulto) [inline rmotr_sis/templates/courses/lecture_detail.html]: I'm repeating this on purpose. Once we know it's good we can refactor.

File: rmotr_sis/assignments/models.py [hunk 2]
```diff
@@ -35,11 +35,13 @@ def get_status(self, student):
...
         qs = AssignmentAttempt.objects.filter(student=student, assignment=self)
-        if qs.count() == 0:
+        count = qs.count()
+        if count == 0:
             return 'pending'
-        qs = qs.filter(resolved=True)
-        if qs.exists():
+        if qs.filter(resolved=True).exists():
             return 'resolved'
+        if count == 1 and qs.filter(execution_time=None):
```
Comment (martinzugnoni) [inline rmotr_sis/assignments/models.py]: I'm not sure if we need a `.exists()` at the end of `qs.filter(execution_time=None)`. In the same way we do for `qs.filter(resolved=True).exists()`.

Comment (santiagobasulto) [inline rmotr_sis/assignments/models.py]: +1

File: rmotr_sis/templates/courses/lecture_detail.html [hunk 3]
```diff
@@ -93,7 +102,15 @@
                 <tr>
                     <td>{{ student.get_full_name }}</td>
                     {% for assignment, status in assignments.items %}
-                    <td>{{ status|capfirst }}</td>
+                    <td>
+                        {% if status == 'resolved' %}
+                          <i class="fa fa-check" style="color: #5cb85c"></i>
+                        {% elif status == 'failed' %}
+                          <i class="fa fa-times" style="color: red"></i>
+                        {% elif status == 'pending' or status == 'unsubmitted'%}
```
Comment (martinzugnoni) [inline rmotr_sis/templates/courses/lecture_detail.html]: We wouldn't notice visually the difference between "pending" and "unsubmitted". Maybe we can add the status as `title` attribute of the `td` tag, or similar.

**Comment by martinzugnoni** [comment]
+1

**Pull request merged**